### PR TITLE
Added Maven Integration!

### PR DESCRIPTION
**Pull request by dhagandev**
Removed the twitter4j jar and verified that it can be run in terminal!

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,18 +1,47 @@
...
-            <!--<version>[4.0,)</version>-->
-        <!--</dependency>-->
-    <!--</dependencies>-->
-<!--</project>-->
\ No newline at end of file
+    <dependencies>
+        <dependency>
+            <groupId>org.twitter4j</groupId>
+            <artifactId>twitter4j-core</artifactId>
+            <version>[4.0,)</version>
```
Comment (kev-chan) [inline pom.xml]: move version to a properties section

File: src/main/java/META-INF/MANIFEST.MF [hunk 2]
```diff
@@ -1,3 +1,3 @@
 Manifest-Version: 1.0
```
Comment (kev-chan) [inline src/main/java/META-INF/MANIFEST.MF:1]: is manifest file still needed with maven?

**Review by kev-chan — CHANGES_REQUESTED** [review]

File: src/main/java/META-INF/MANIFEST.MF (hunk 2 above)
Comment (dhagandev) [reply src/main/java/META-INF/MANIFEST.MF:1]: Needed no, but can be used with Maven still. From what I saw, you can either declare <manifest/> and fill out the variables needed and such, or declare <manifestFile/> and have it point to your pre-existing manifest file.

File: README.md [hunk 3]
```diff
@@ -7,19 +7,17 @@ Install Java 8. We recommend installing IntelliJ to edit files, but any editor s
...
+2. Copy and paste this file into your home directory; rename it ```hardcoded_keys.xml```
 3. Fill in the fields for the consumerKey, consumerSecret, accessToken, and accessSecret within ```hardcoded_keys.xml```.
 
 ### Create the JAR File ###
 4. Open a terminal shell and navigate to the LiUni_Onboarding project.
-5. Run ```cd src/main/java```.
-6. Run ```javac -cp ../../../twitter4j-core-4.0.4.jar liuni/*.java``` to compile the project.
-7. Run ```jar cfm ../../../out/artifacts/LiUni_Onboarding_jar/LiUni_Onboarding.jar META-INF/MANIFEST.MF liuni/*.class```
-8. Move your ```hardcoded_keys.xml``` to ```out/artifacts/LiUni_Onboarding_jar```
+5. Run ```mvn package`` to create the jar.
```
Comment (kev-chan) [inline README.md]: fix the spacing on the readme

**Review by kev-chan — CHANGES_REQUESTED** [review]

**Review by kev-chan — APPROVED** [review]

**Pull request merged**